### PR TITLE
Add TypeScript definitions

### PR DESCRIPTION
**Pull request by josh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "version": "1.0.2",
   "main": "index.umd.js",
   "module": "index.js",
+  "types": "index.d.ts",
   "files": [
```
Comment (muan) [inline package.json]:
```suggestion
   "files": [
     "index.d.ts",
```

Comment (josh) [reply package.json]: :D

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "version": "1.0.2",
   "main": "index.umd.js",
   "module": "index.js",
-  "files": [
+  "types": "index.d.ts",
+   "files": [
```
Comment (muan) [inline package.json]: 🤷🏻‍♀️

```suggestion
  "files": [
```

Comment (josh) [reply package.json]: Whitespace is hard.

**Comment by se-bastiaan** [comment]
I believe that this is invalid Typescript: https://www.typescriptlang.org/play/index.html#code/KYDwDg9gTgLgBAE2AYwDYEMrEcAZugV1XlwIDtkYBLCMuXaAWwBF0Z0BRMmKK4AZwAUDKIwBccABIAVALIAZAGJMOqYI2DcAlBICCUKOgCeAHgDa-HlTIBzADRxLvWwF0AfAG4AUEA.

This should work:
```
declare function formDataEntries(form: HTMLFormElement): Array<[string, string]>;
export default formDataEntries;
```

**Pull request merged**